telemetry/telemetry.go

```diff
@@ -7,12 +7,16 @@ import (
 	"encoding/json"
 
 	"github.com/Azure/azure-container-networking/aitelemetry"
+	zapLog "github.com/Azure/azure-container-networking/cni/log"
 	"github.com/Azure/azure-container-networking/common"
 	"github.com/Azure/azure-container-networking/log"
 	"github.com/Azure/azure-container-networking/platform"
 	"github.com/pkg/errors"
+	"go.uber.org/zap"
 )
 
+var logger = zapLog.CNILogger.With(zap.String("component", "cni-telemetry"))
```

Comment:
Isn't logger passed as argument to NewTelemetryBuffer ? where this logger is used?